CHANGELOG.md

```diff
@@ -13,15 +13,30 @@ All notable changes to this library are documented in this file.
 - BFV: replaced `bfv.Evaluator.InnerSum` with the more complete `rlwe.Evaluator.InnerSum`.
 - BFV: the `Evaluator` addition and subtraction no longer enforce BFV-specific operand types.
 - BFV: the maximum degree allowed for ciphertext multiplication has been reduced to two (same as `bgv` and `ckks`).
+- BFV: removed checks during addition and subtraction for the type of plaintext.
 - CKKS: added the `Polynomial.Lazy` field which specifies if the power basis is computed with lazy relinearization.
 - CKKS: made `NttAndMontgomery` thread safe again!
 - CKKS: removed `Evaluator` methods `MultByGaussianInteger`, `MultByGaussianIntegerThenAdd`, `MultByi`, `MultByiNew`, `DivByi` and `DivByiNew`. These are now all handled by the methods `MultByConst[...]`.
 - CKKS: updated the behavior of `MultByConstAndAdd`.
 - CKKS: fixed the median statistics of `PrecisionStats`, that were off by one index. 
+- CKKS: renamed the `Parameters` field `DefaultScale` to `LogScale`, which now takes a value in log2.
+- CKKS: the `Parameters` field `LogSlots` now has a default value which is the maximum number of slots possible for the given parameters.
+- CKKS: variable `BSGSRatio` is now `LogBSGSRatio` and is given in log2.
+- CKKS/Bootstrapping: complete refactoring the bootstrapping parameters for better usability.
+- CKKS/Bootstrapping: upon bootstrapping, the method will check that the ciphertext scale is a power of two.
+- CKKS/Bootstrapping: added the iterative bootstrapping `META-BTS` of `Youngjin et al.`.
```

Comment:
[Youngjin et al.](<eprint link>) instead of  `Youngjin et al.`?

Reply:
done

Reply:
Remove the quotes so that the link is rendered correctly (see [current rendering](https://github.com/tuneinsight/lattigo/blob/dev_bootstrapping_2.0/CHANGELOG.md))

Reply:
Thank's, done